lib/ui/podcast/comment_List.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:anytime/bloc/comments/comments_bloc.dart';
+import 'package:anytime/entities/comments.dart';
+import 'package:anytime/ui/podcast/comment_child.dart';
+import 'package:flutter/material.dart';
+
+import 'package:nostr_tools/nostr_tools.dart';
+
+import '../widgets/delayed_progress_indicator.dart';
+
+class CommentRender extends StatefulWidget {
```

Comment:
Name does not match file name. File name has capital letter.
Please align them.